test-scripts/master.py

```diff
@@ -203,11 +203,9 @@ def setup_cluster(tconf, hosts, ips, conf, nocon):
     _confs = dict(
         [(_h, {'ns': 0, 'nm': _mph, 'svrs': [], 'moms': []}) for _h in hosts])
     _hi = 0
-    mom_cnt_per_svr = 0
-    if _ts > _hl:
-        # if no of servers are more than no of hosts
-        # allot moms to servers
-        mom_cnt_per_svr = int(_tm / _ts)
+    # if no of servers are more than no of hosts
+    # allot moms to servers
+    mom_cnt_per_svr = int(_tm / _ts)
```

Comment:
the comment doesn't seem correct anymore as you removed the check for _ts > _hl ?